#xy position of classified and idenified obstacle relative to base_link

# passed atributes from darknet_ros bounding box
float64 probability
int16 id
string Class 

# timestamped point of obstacle generated by cv_lidar_fusion_node (center of bounding box) 
geometry_msgs/PointStamped point 
